CMakeLists.txt

```diff
@@ -39,6 +42,10 @@ else()
 	project(openage CXX)
 endif()
 
+# C++ standard
+set(openage_cxx_standard 14)
```

Comment:
This variable and its usage should be implied by the following lines from `HandleCXXOptions.cmake`:
```cmake
set(CMAKE_CXX_STANDARD 14)
set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Reply:
ah, i changed this so every target has to declare its cxx_standard, so the standard definition in `HandleCXXOptions.cmake` is obsolete.

Reply:
This approach looks more verbose. Any particular reason to not use CMake's builtin variable(s)?

Reply:
I wanted a single location where our standard is specified; that variable is then used for cython modules and the libopenage.

Reply:
All cython modules and libopenage are added using `add_library` and should respect the value in `CMAKE_CXX_STANDARD` automatically.
Using a different variable has the possibility of someone missing the specification for a new target.

Reply:
You're right. I'll try if it still works, for nyan i had to specify it for the library because the include mechanism does `add_subdirectory` on the nyan repo, so the c++17 has to be specified for the nyan targets separately. Let's see if it still works with the global definition.

Reply:
For nyan (assuming the commit removing `set(CMAKE_CXX_STANDARD 17)` is reverted), it should be okay to use  `add_subdirectory`, as CMake keeps that definition limited to the nyan directory scope.

Reply:
hm i'm not sure what my mistake was, then. I added the global cxx standard setting in openage back in.